ocs_ci/ocs/resources/pod.py

```diff
@@ -498,6 +498,23 @@ def get_csi_provisioner_pod(interface):
     return provisioner_pod
 
 
+def get_csi_snapshotner_pod():
```

Comment:
typo in snapshotner? shouldn't it be `snapshoter`

Reply:
fixed.